.github/workflows/ci.yml

```diff
@@ -97,6 +97,26 @@ jobs:
         uses: pantsbuild/actions/run-tox@e63d2d0e3c339bdffbe5e51e7c39550e3bc527bb
         with:
           tox-env: py${{ matrix.python-version[0] }}${{ matrix.python-version[1] }}-pip${{ matrix.pip-version }}
+  cpython-unit-tests-aarch64:
```

Comment:
Because these run on a self-hosted runner with Pythons already set up just-so, the setup, expose and cache interpreter steps are not relevant, so these are separate jobs rather than matrix entries in the existing ones.